src/content/en/updates/2018/03/smooshgate.md

```diff
@@ -0,0 +1,217 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: Recently the internet was on fire because of #SmooshGate. What is it about, and where do we go from here? This write-up gives an overview.
+
+{# wf_updated_on: 2018-03-19 #}
+{# wf_published_on: 2018-03-19 #}
+{# wf_tags: javascript #}
+{# wf_featured_image: /web/updates/images/generic/info.png #}
+{# wf_featured_snippet: Recently the internet was on fire because of #SmooshGate. What is it about, and where do we go from here? This write-up gives an overview. #}
```

Comment:
I read this snippet as something that could throw further fuel on the fire. Can we change it up a bit?

Reply:
How about

> What can #SmooshGate teach us about standards development and the Web Platform? This write-up gives an overview.